From phil@ipom.com Wed Oct 12 21:02:40 2005
Message-ID: <434DDB47.2000802@ipom.com>
Date: Wed, 12 Oct 2005 20:57:59 -0700
From: Phil Dibowitz <phil@ipom.com>
To: Greg KH <greg@kroah.com>, 
 USB Storage list <usb-storage@lists.one-eyed-alien.net>,
 USB development list <linux-usb-devel@lists.sourceforge.net>, 
 Alan Stern <stern@rowland.harvard.edu>
Subject: USB Storage: Another unusual_devs entry

From: Alan Stern <stern@rowland.harvard.edu>

This unusual_devs.h entry fixes kernel bug #5231.  Debugging logs and
/proc/bus/usb/devices listing are available as part of the bug report.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |    7 +++++++
 1 file changed, 7 insertions(+)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h
@@ -113,6 +113,13 @@ UNUSUAL_DEV(  0x0419, 0xaaf6, 0x0100, 0x
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
 		US_FL_IGNORE_RESIDUE ),
 
+/* Reported by Florian Mickler <fmickler@gmx.de> */
+UNUSUAL_DEV(  0x419, 0xaaf6, 0x0100, 0x0100,
+		"TrekStor",
+		"i.Beat Joy 2.0",
+		US_SC_DEVICE, US_PR_DEVICE, NULL,
+		US_FL_IGNORE_RESIDUE ),
+
 /* Reported by Olaf Hering <olh@suse.de> from novell bug #105878 */
 UNUSUAL_DEV(  0x0424, 0x0fdc, 0x0210, 0x0210,
 		"SMSC",
